packages/operation-execution/src/index.ts

```diff
@@ -404,7 +404,18 @@ export class Operation<I, O> implements IOperation<I, O> {
     const output = parseParameters(this._operationDefinition, use, input);
     return output as InputOutput<I, O>[Use];
   }
+
   parseToParameters(use: "in" | "out", input: I | O): Parameters {
+    if (
```

Comment:
Not parsing on edgecase of single return parameter. Should fix now with hardcoded setting to return object.